src/DataTable/DataTable.features.stories.tsx

```diff
@@ -1656,3 +1656,30 @@ export const WithNetworkError = () => {
     </Table.Container>
   )
 }
+
+export const StressTest = () => (
```

Comment:
Some stories have a "Dev" folder for stories like this. I think that makes sense for this story too.